### PR TITLE
Added image Type field to the image details slide panel

### LINKED ISSUE #1095: Image "Type" information is not available in Details page
### Preconditions (*)
Magento 2.4-develop
Adobe-Stock-Integration 1.1-develop

1. Installed Magento with Adobe Stock Integration
2. Configured integration in Stores -> Configuration -> Advanced-> System -> Adobe Stock Integration fieldset
3. Enable Enhanced Media Gallery in Stores -> Configuration -> Advanced -> System 
enhansedMG
4. An image added to Media Gallery from Adobe Stock

### Steps to reproduce (*)
1. From Admin go to **Content** - **Pages**, click **Add New Page**
2. Expand **Content**,click **Show / Hide Editor**, click **Insert Image...**
3. Select an image added from Adobe Stock 
4. Click on three dots near the image and select **View Details**
three_dots
5. Verify _Type_ information

### Expected result (*)
The _Type_ information should be displayed in the image details

### Actual result (*)
The _Type_ information is missing from Image Details
type

### PR DESCRIPTION
**Pull request by engcom-Charlie**
### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#1095: Image "Type" information is not available in Details page
2. ...

### Manual testing scenarios (*)

Please see https://github.com/magento/adobe-stock-integration/issues/1095

### CODE REVIEW

**Comment by engcom-Charlie** [comment]
@magento run all tests

File: AdobeStockImage/etc/db_schema.xml [hunk 1]
```diff
@@ -8,5 +8,6 @@
 <schema xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Setup/Declaration/Schema/etc/schema.xsd">
     <table name="media_gallery_asset_grid" resource="default" engine="innodb" comment="Media Gallery Asset">
         <column xsi:type="boolean" name="licensed" comment="Licensed image"/>
+        <column xsi:type="int" name="media_type_id" padding="10" unsigned="true" nullable="false" identity="false" default="0" comment="Media Type Id"/>
```
Comment (sivaschenko) [inline AdobeStockImage/etc/db_schema.xml]: There is no need to add the media_type_id to the grid. This attribute is not displayed in the grid, only in the image details panel

File: MediaGalleryUi/Model/GetImageDetailsByAssetId.php [hunk 2]
```diff
@@ -109,6 +119,10 @@ public function execute(int $assetId): array
             'title' => $asset->getTitle(),
             'image_id' => $assetId,
             'details' => [
+                [
+                    'title' => __('Type'),
+                    'value' => $this->getTypeImageByMediaTypeId($mediaTypeId),
```
Comment (sivaschenko) [inline MediaGalleryUi/Model/GetImageDetailsByAssetId.php]: Please retrieve the type from the contenty_type field of the media_gallery_asset.

If the asset comes from Adobe Stock (there is information stored in adobe_stock_asset) - please update this information in `\Magento\AdobeStockImageAdminUi\Plugin\AddAdobeStockImageDetailsPlugin`

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for the pull request @engcom-Charlie ! Please take a look at my review comments

**Review by sivaschenko — APPROVED** [review]

**Pull request merged**